apps/new-dealer/models/submission.js

```diff
@@ -2,10 +2,27 @@
 /* eslint complexity: 0 max-statements: 0 */
 const contains = (arr, val) => arr.indexOf(val) > -1 ? 'Yes' : 'No';
 
+const authorityType = usage => {
+  if (usage.indexOf('arm-guards') > -1) {
+    return 'Maritime Guards';
+  }
+
+  if (usage.indexOf('transport') > -1 || usage.indexOf('transfer') > -1) {
```

Comment:
TBH, since we're deploying on `node@6.9.x` I'd just go for `usage.includes('...')`.

I should get into the habit of doing that.